packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEdit.tsx

```diff
@@ -65,6 +65,10 @@ export const StocktakeLineEdit: FC<StocktakeLineEditProps> = ({
     }
   };
 
+  const hasValidBatches = draftLines.some(
```

Comment:
prevent saving when there are no valid batches.. which I _think_ was the problem raised